### PR TITLE
Feature/profile

### PR DESCRIPTION
**Pull request by h4l0anne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,60 @@
...
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql)
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Anne Li . Jarvis Consulting
 
+I recently graduated from Ryerson University where I received my Bachelor of Engineering, majoring in Computer Engineering. I delved into programming during my time at university and I am fascinated by its endless possibilities in solving problems. Previously I did a 4-month internship at De Havilland, an aircraft manufacturing company, where I worked as a software engineer to help build their data-visualization tool. Currently, I am a data engineer trainee at Jarvis where I have been able to challenge myself with various projects utilizing Linux/Bash, Java 8, Docker, Maven, SpringBoot, GCP (and MORE) in a collaborative environment. I keep myself open to trying new things; whether that be my technical abilities, in personal projects, or providing quality service, I'm always eager to continuously learn to be a better engineer.
```
Comment (jarviscanada) [inline README.md:3]: software and data engineer

Comment (jarviscanada) [reply README.md:3]: collaborative environment
agile/scrum env

File: README.md [hunk 2]
```diff
@@ -1,8 +1,60 @@
...
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** C, JavaScript, Python, HTML, CSS
```
Comment (jarviscanada) [inline README.md:9]: Springboot, Maven, docker (competent)

File: README.md [hunk 3]
```diff
@@ -1,8 +1,60 @@
...
+
+**Familiar:** Docker, GCP, Pub/Sub, REST API, Spring, Django/Flask, Matlab
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AnneLi](https://github.com/jarviscanada/jarvis_data_eng_AnneLi)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/linux_sql)]: This project automates the process of monitoring servers/nodes of linux that are connected through a switch and communicates internally through IPv4 addresses. Implemented bash scripts that obtain data on hardware specifications and on-going Linux resource usage data.
```
Comment (jarviscanada) [inline README.md:18]: Developed an application in bash that ...

Bash, Linux, Docker, RDBMS/PostgreSQL, SQL, crontab

File: README.md [hunk 4]
```diff
@@ -1,8 +1,60 @@
...
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/core_java)]:
+      
+  - Twitter App: Implemented a CLI app that post|show|delete tweets to the user account from the command-line via Twiiter REST API using the MVC architecture.
```
Comment (jarviscanada) [inline README.md:22]: java 8, maven, junit & mockito, Intellij, docker, common librareis (HTTP, JSON, OAtuh),

design -> implementaion/tech -> test -> deloloy

File: README.md [hunk 5]
```diff
@@ -1,8 +1,60 @@
...
+  - JDBC App: This project sets up a connection to the PostgreSQL database by leveraging the JDBC API that manages customer and order records.
+  - Grep App: This app echoes partial functionality of the linux command `grep`. The app searches for a text pattern recursively through all the files in a given directory and outputs the matched lines to a designated file.
```
Comment (jarviscanada) [inline README.md:24]: Java 8, Lamnbda, I/O interface, regex

File: README.md [hunk 6]
```diff
@@ -1,8 +1,60 @@
...
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/hadoop)]: Not Started
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/spark)]: In-progress
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/cloud_devops)]: In-progress
+
+
+## Highlighted Projects
+**SERVE.AI (Web application)**: Developed an Online Cafeteria Application utilizing the Django framework, SQLite3, Bootstrap using Agile methodologies and practices. Practiced SDLC process models, requirements elicitation, system modelling, project management, teamwork, navigating documentations and software testing.
```
Comment (jarviscanada) [inline README.md:38]: Python/Django

File: README.md [hunk 7]
```diff
@@ -1,8 +1,60 @@
...
+
+**SMART CITY APPLICATION**: Learned about distributed System and networking technology by designing and developing an urban IoT website which generates pickup schedule based on real-time capacity of sensor-attached garbage bin. Programmed ESP8266 using C++ in Arduino IDE to send JSON data via MQTT protocol to Raspberry Pi.
```
Comment (jarviscanada) [inline README.md:40]: Developed a web application in {language/tech} that detect blah.. using senor ....
hardware, software, method/algo,

File: README.md [hunk 8]
```diff
@@ -1,8 +1,60 @@
...
+
+
+## Professional Experiences
+
+**Data Engineer, Jarvis (Jan 2021-present)**: Implemented projects that solves Big-data problems utilizing a set of tools and technologies such as Linux, SQL, Core Java, SpringBoot, DevOps/Cloud, Hadoop and Spark/Scala. Conducted and participated in daily scrum meetings in a bi-weekly sprint.
```
Comment (jarviscanada) [inline README.md:45]: Conducted and participated in daily scrum meetings in a bi-weekly sprint. _> talk more about agile

File: README.md [hunk 9]
```diff
@@ -1,8 +1,60 @@
...
+
+**Coding Instructor, Codezilla (Jun 2020-Dec 2020)**: Led summer camp programs on game development and scratch coding to help youths develop problem-solving skills.
```
Comment (jarviscanada) [inline README.md:47]: soft skills 
commuition, leadership, collaberation, etc..

File: README.md [hunk 10]
```diff
@@ -1,8 +1,60 @@
...
+
+**Software Engineer Intern, De Havilland (May 2019-August 2019)**: Developed a data visualization tool to analyze performance efficiently using PowerBI and VBA. Contributed to automation projects and development of technology roadmaps.
```
Comment (jarviscanada) [inline README.md:49]: data type, csv, excel, json
more details

File: core_java/grep/Dockerfile [hunk 11]
```diff
@@ -0,0 +1,3 @@
+FROM openjdk:8-alpine
```
Comment (jarviscanada) [inline core_java/grep/Dockerfile:1]: remove core_java/grep/.pom.xml.swp

File: .gitignore [hunk 12]
```diff
@@ -0,0 +1,9 @@
+target/
+*.class
```
Comment (jarviscanada) [inline .gitignore:2]: git checkout master
git pull
git checkout -b feature/profile_2

commission only profile files

make another PR

**Review by jarviscanada — APPROVED** [review]
fix all and create new branch and merge

**Comment by ch33sebytes** [comment]
thank you for the feedbacks, incorporated in feature/profile_2
